documentation/specs/static-graph.md

```diff
@@ -1,20 +1,18 @@
 - [Static Graph](#static-graph)
   - [Overview](#overview)
     - [Motivations](#motivations)
-    - [M1 - MVP](#m1---mvp)
```

Comment:
I removed the timelines and such since they ended up not really mapping well to the actual order we were working on things and the GitHub projects were a much better way to track and plan.